src/utils/bridgeMonitor.js

```diff
@@ -0,0 +1,169 @@
+const config = require('config');
+const axios = require('axios');
+const cron = require('node-cron');
+const logger = require('winston');
+const { AdminTypes } = require('../models/pledgeAdmins.model');
+const { DonationStatus, DONATION_BRIDGE_STATUS } = require('../models/donations.model');
+const { ZERO_ADDRESS } = require('../blockchain/lib/web3Helpers');
+
+const bridgeMonitorBaseUrl = config.get('bridgeMonitorBaseUrl');
+const getDonationStatusFromBridge = async ({ txHash }) => {
+  /**
+   * If you want to see an example of bridge monitor response you can check this link
+   * @see{@link https://feathers.bridge.beta.giveth.io/payments?event.returnValues.reference=0xe5f676f6d24fbe43feddb80dcebbedf86e35835b1c5dd984cda46faff73dd98a}
+   * @type {*}
+   */
+  const result = await axios.get(`${bridgeMonitorBaseUrl}/payments`, {
+    params: {
+      'event.returnValues.reference': txHash,
+    },
+  });
+  if (result.data && result.data.data && result.data.data.length >= 0) {
+    return result.data.data[0];
+  }
+  return undefined;
+};
+
+const updateMilestoneStatusToPaid = async (app, donation) => {
+  const donationService = app.service('donations');
+  const milestoneService = app.service('milestones');
+  const milestone = await milestoneService.get(donation.ownerTypeId);
+  const { maxAmount, reviewerAddress, fullyFunded } = milestone;
+
+  // never set uncapped or without-reviewer non-fullyFunded milestones as PAID
+  const hasReviewer = reviewerAddress && reviewerAddress !== ZERO_ADDRESS;
+  if (!maxAmount || (!fullyFunded && !hasReviewer)) return;
```

Comment:
We have checked this conditions and made milestone PAID, so just check the status is PAID

Reply:
Check only milestone is PAID and all of its donations has been paid on bridge

Reply:
done